astropy/modeling/polynomial.py

```diff
@@ -322,7 +322,15 @@ def prepare_inputs(self, x, y, **kwargs):
 
 class Chebyshev1D(PolynomialModel):
     """
-    1D Chebyshev polynomial of the 1st kind.
+    Univariate Chebyshev series.
+
+    It is defined as:
+
+    .. math::
+
+        P(x) = \sum_{i=0}^{i=n}C_{i} * T_{i}(x)
```

Comment:
(or have it start as ``r"""``, which I slightly favor because its a bit easier to read)